Packs/Core/ReleaseNotes/3_0_72.md

```diff
@@ -0,0 +1,32 @@
+
+#### Playbooks
+
+##### New: Event Log Was Cleared
+
+New: This playbook is designed to handle the following alerts:  
+Windows Event Log was cleared using wevtutil.exe
+Security Event Log was cleared using wevtutil.exe
+A Sensitive Windows Event Log was cleared using wevtutil.exe
+Windows event logs were cleared with PowerShell
+Suspicious clear or delete security provider event logs with PowerShell
+Suspicious clear or delete default providers event logs with PowerShell
+Windows event logs cleared using wmic.exe
+
+The playbook executes the following stages:
+
+Investigation:
+Check the following parameters to determine if remediation actions are needed:
+XDR alerts related to the hostname by MITRE tactics indicating malicious activity.
+Whether the process is signed.
```

Comment:
```suggestion
Whether the action generated process is signed or not.
```

Reply:
Done. changed to "Whether the CGO or the OSParent process is unsigned."